### PR TITLE
[PennyPalooza] update `pyproject.toml` project description

### PR DESCRIPTION
**Pull request by andrijapau**
Updates the project description to be more in line with company's mission.

### CODE REVIEW

**Review by CatalinaAlbornoz — APPROVED** [review]
Thanks!

File: pyproject.toml [hunk 1]
```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "pennylane"
-description = "PennyLane is a cross-platform Python library for quantum computing, quantum machine learning, and quantum chemistry. Train a quantum computer the same way as a neural network."
+description = "PennyLane is a cross-platform Python library for quantum computing, quantum chemistry and quantum machine learning."
 readme = "README.md"
```
Comment (sardorbekergashev321-source) [inline pyproject.toml:8]: # pip install torch torchvision
import torch
from torch import nn, optim
from torchvision import datasets, transforms
from torch.utils.data import DataLoader

# настройки
batch_size = 64
lr = 1e-3
epochs = 5
device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')

# данные
transform = transforms.Compose([
    transforms.ToTensor(),
    transforms.Normalize((0.1307,), (0.3081,))
])

train_ds = datasets.MNIST(root='mnist_data', train=True, download=True, transform=transform)
test_ds  = datasets.MNIST(root='mnist_data', train=False, download=True, transform=transform)
train_loader = DataLoader(train_ds, batch_size=batch_size, shuffle=True)
test_loader  = DataLoader(test_ds, batch_size=1000, shuffle=False)

# модель
class SimpleNet(nn.Module):
    def __init__(self):
        super().__init__()
        self.net = nn.Sequential(
            nn.Flatten(),
            nn.Linear(28*28, 256),
            nn.ReLU(),
            nn.Dropout(0.2),
            nn.Linear(256, 10)
        )
    def forward(self, x):
        return self.net(x)

model = SimpleNet().to(device)
criterion = nn.CrossEntropyLoss()
optimizer = optim.Adam(model.parameters(), lr=lr)

# обучение
for epoch in range(1, epochs+1):
    model.train()
    total_loss = 0
    for X, y in train_loader:
        X, y = X.to(device), y.to(device)
        optimizer.zero_grad()
        logits = model(X)
        loss = criterion(logits, y)
        loss.backward()
        optimizer.step()
        total_loss += loss.item() * X.size(0)
    avg_loss = total_loss / len(train_loader.dataset)
    # валидация
    model.eval()
    correct = 0
    with torch.no_grad():
        for X, y in test_loader:
            X, y = X.to(device), y.to(device)
            preds = model(X).argmax(dim=1)
            correct += (preds == y).sum().item()
    acc = correct / len(test_loader.dataset)
    print(f"Epoch {epoch}: train_loss={avg_loss:.4f}, test_acc={acc:.4f}")

# сохранить модель
torch.save(model.state_dict(), "simple_mnist.pth")

Comment (sardorbekergashev321-source) [reply pyproject.toml:8]: # pip install torch torchvision
import torch
from torch import nn, optim
from torchvision import datasets, transforms
from torch.utils.data import DataLoader

# настройки
batch_size = 64
lr = 1e-3
epochs = 5
device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')

# данные
transform = transforms.Compose([
    transforms.ToTensor(),
    transforms.Normalize((0.1307,), (0.3081,))
])

train_ds = datasets.MNIST(root='mnist_data', train=True, download=True, transform=transform)
test_ds  = datasets.MNIST(root='mnist_data', train=False, download=True, transform=transform)
train_loader = DataLoader(train_ds, batch_size=batch_size, shuffle=True)
test_loader  = DataLoader(test_ds, batch_size=1000, shuffle=False)

# модель
class SimpleNet(nn.Module):
    def __init__(self):
        super().__init__()
        self.net = nn.Sequential(
            nn.Flatten(),
            nn.Linear(28*28, 256),
            nn.ReLU(),
            nn.Dropout(0.2),
            nn.Linear(256, 10)
        )
    def forward(self, x):
        return self.net(x)

model = SimpleNet().to(device)
criterion = nn.CrossEntropyLoss()
optimizer = optim.Adam(model.parameters(), lr=lr)

# обучение
for epoch in range(1, epochs+1):
    model.train()
    total_loss = 0
    for X, y in train_loader:
        X, y = X.to(device), y.to(device)
        optimizer.zero_grad()
        logits = model(X)
        loss = criterion(logits, y)
        loss.backward()
        optimizer.step()
        total_loss += loss.item() * X.size(0)
    avg_loss = total_loss / len(train_loader.dataset)
    # валидация
    model.eval()
    correct = 0
    with torch.no_grad():
        for X, y in test_loader:
            X, y = X.to(device), y.to(device)
            preds = model(X).argmax(dim=1)
            correct += (preds == y).sum().item()
    acc = correct / len(test_loader.dataset)
    print(f"Epoch {epoch}: train_loss={avg_loss:.4f}, test_acc={acc:.4f}")

# сохранить модель
torch.save(model.state_dict(), "simple_mnist.pth")

**Review by sardorbekergashev321-source — APPROVED** [review]

**Comment by andrijapau** [comment]
Closing as we are not sure of our messaging internally.
